chord_metadata_service/mohpackets/data/data_converter.py

```diff
@@ -141,29 +142,40 @@ def replace_values(input_data, transformation_rules):
     return input_data
 
 
-def main():
-    print("Select an option:")
-    print("1. Convert small dataset")
-    print("2. Convert medium dataset")
-    print("3. Convert large dataset")
-    print("4. Exit")
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--size',
+        type=str,
+        default='s',
+        choices=['s', 'm', 'l'],
+        help="Size of the dataset to convert (default: small)"
```

Comment:
a bit of extra info for the user
```suggestion
        help="Size of the dataset to convert, options: 's' for small, 'm' for medium, 'l' for large (default: small)"
```

Reply:
Fixed.